.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: false
 
 env: PYTHONDONTWRITEBYTECODE=x
 
+ghc: 7.8
```

Comment:
I think I was the one who added this, but it's actually not needed - we ended up installing GHC ourself (because that's how this sort of thing inevitably goes in Hypothesis build land. Sigh) using stack.